src/css/main.less

```diff
@@ -395,45 +395,43 @@ a:focus {
 
       .labels {
         .label {
+          display: none;
+          overflow: hidden;
+          text-overflow: ellipsis;
+          max-width: @app-list-label-width;
+          white-space: nowrap;
+          padding: @horizontal-spacing-unit*0.25 @base-spacing-unit*1.5;
           margin-left: @horizontal-spacing-unit;
-        }
-      }
-    }
-  }
-
-  .label {
-    display: none;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    max-width: @app-list-label-width;
-    white-space: nowrap;
-    padding: @horizontal-spacing-unit*0.25 @base-spacing-unit*1.5;
-    background-color: @sidebar-bg-color;
-    border-radius: @base-spacing-unit*2;
-    color: @navbar-inactive-color;
-    font-size: @small-font-size;
-    font-weight: normal;
-
-    &.more {
-      display: inline-block;
-    }
-  }
+          background-color: @sidebar-bg-color;
+          border-radius: @base-spacing-unit*2;
+          color: @navbar-inactive-color;
+          font-size: @small-font-size;
+          font-weight: normal;
```

Comment:
Sorry, didn't know that... 😇 I'll sort them next time. `recess` sounds great. We should give it a try!